src/javascript/app/pages/cashier/account_transfer.js

```diff
@@ -56,8 +57,15 @@ const AccountTransfer = (() => {
         }
         if (fragment_transfer_to.childElementCount > 1) {
             el_transfer_to.innerHTML = fragment_transfer_to.innerHTML;
+            el_transfer_to.onchange = () => {
+                const to_currency = el_transfer_to.options[el_transfer_to.selectedIndex].getAttribute('data-currency');
+                el_transfer_info.setVisibility(client_currency === to_currency ? 0 : 1);
```

Comment:
why not `el_transfer_info.setVisibility(client_currency !== to_currency);`

Reply:
I wasn't aware it accepted boolean values